.github/workflows/build-lint-test.yml

```diff
@@ -30,11 +43,11 @@ jobs:
             ~/.cargo/registry/index/
             ~/.cargo/registry/cache/
             ~/.cargo/git/db/
-            target/
+            ./backend/target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
           restore-keys: ${{ runner.os }}-cargo-
       - name: Install sqlx-cli
-        run: cargo install sqlx-cli --no-default-features --features sqlite
+        run: cargo --verbose --locked install sqlx-cli --no-default-features --features sqlite
```

Comment:
This can be removed to cut down build time by also using sqlx offline here.

Reply:
Fixed in [5bcf348](https://github.com/kiesraad/abacus/pull/429/commits/5bcf348fc5def4b7c545d1e5234625595c110e93)